stories/wizard/wizard.stories.js

```diff
@@ -0,0 +1,868 @@
+                <div class="fd-wizard__step-wrapper">
+                    <a class="fd-wizard__step-container" tabindex="0" aria-label="Products" aria-disabled="true">
+                        <span class="fd-wizard__step-indicator sap-icon--product"></span>
+                        <div class="fd-wizard__label-container fd-wizard__label-container--optional">
+                            <span class="fd-wizard__label">Products</span>
+                            <span class="fd-wizard__optional-text">(Optional)</span>
+                        </div>
+                    </a>
+                    <span class="fd-wizard__connector"></span>
+                </div>
+            </li>
+            <li class="fd-wizard__step fd-wizard__step--upcoming fd-wizard__step--stacked">
+                <div class="fd-wizard__step-wrapper">
+                    <a class="fd-wizard__step-container" tabindex="0" aria-label="Additional Information" aria-disabled="true">
+                        <span class="fd-wizard__step-indicator sap-icon--course-program"></span>
+                        <div class="fd-wizard__label-container">
+                            <span class="fd-wizard__label">Additional Information</span>
+                        </div>
+                    </a>
+                    <span class="fd-wizard__connector"></span>
+                </div>
+            </li>
+        </ul>
+    </nav>
+    <section class="fd-wizard__content fd-wizard__content--xl" id="wizard-section-5" style="min-height: 300px;">
+        <div>
+            Sed fermentum, mi et tristique ullamcorper, sapien sapien faucibus sem, quis pretium nibh lorem malesuada diam. Nulla quis arcu aliquet, feugiat massa semper, volutpat diam. Nam vitae ante posuere, molestie neque sit amet, dapibus velit. Maecenas eleifend tempor lorem. Mauris vitae elementum mi, sed eleifend ligula. Nulla tempor vulputate dolor, nec dignissim quam convallis ut. Praesent vitae commodo felis, ut iaculis felis. Fusce quis eleifend sapien, eget facilisis nibh. Suspendisse est velit, scelerisque ut commodo eget, dignissim quis metus. Cras faucibus consequat gravida. Curabitur vitae quam felis. Phasellus ac leo eleifend, commodo tortor et, varius quam. Aliquam erat volutpat
+        </div>
+        <div class="fd-wizard__next-step">
+            <button class="fd-button fd-button--compact fd-button--emphasized">Next Step</button>
+        </div>
+    </section>
+    <footer>
+        <div class="fd-bar fd-bar--page-m_l fd-bar--footer">
+            <div class="fd-bar__right">
+                <div class="fd-bar__element">
+                    <button class="fd-button fd-button--transparent fd-button--compact">Cancel</button>
+                </div>
+            </div>
+        </div>
+    </footer>
+</div>
+`;
+
+responsive.parameters = {
+    docs: {
+        iframeHeight: 950,
+        storyDescription: `If there is enough horizontal space, all labels are shown. 
```

Comment:
If there is insufficient horizontal space, all step labels are displayed. However, when the width decreases, the unselected step labels disappear. To hide the labels, add the <code class="docs-code">fd-wizard\\_\\_step--no-label</code> to the **(?) element/container**.

The unselected steps should stack on top of each other when there is limited screen space. To stack them, add the <code class="docs-code">fd-wizard\\_\\_step--stacked</code> modifier class. The last completed step should display on top, and requires a <code>fd-wizard__step--stacked-top</code> modifier class.

**I don't understand this sentence: No such modifier class is required for the upcoming stacked steps.** Could you clarify this?

Reply:
The stacked items on the left-hand side are completed items, and the ones on the right-hand side are upcoming. You don't need to apply `fd-wizard__step--stacked-top`modifier class on the right-hand (upcoming steps) stack, just for the left-hand (completed steps)

Reply:
Ok so here goes:

"If there is insufficient horizontal space, all step labels are displayed. However, when the width decreases, the unselected step labels disappear. To hide the labels, add the `fd-wizard\_\_step--no-label` to the **(?) element/container**. ------ Add your input here @InnaAtanasova 

The unselected steps should stack on top of each other when there is limited screen space. To stack them, add the `fd-wizard\_\_step--stacked` modifier class. The last completed step should display on top, and requires a `fd-wizard__step--stacked-top` modifier class. Although they are stacked, upcoming steps do not require the stacked-top modifier class. "